### ISSUE TITLE
some potential bugs w.r.t BN layers

### ISSUE THREAD

**Issue by JingyuYang1997** [DuaneNielsen/DeepInfomaxPytorch#18]
Thanks for your sharing! I notice that you use batchnorm layers in  encoder and do not apply model.eval() in testing stage. This may bring performance degradation.

**Comment by DuaneNielsen** [comment]
Thanks for the tip.
